example/main.tf

```diff
@@ -12,21 +13,23 @@ module "minio" {
   container_image                 = "minio/minio:latest" # todo: avoid using tag latest in future releases
   access_key                      = "minio"
   secret_key                      = "minio123"
```

Comment:
Think these can be removed if you want to.

Reply:
Can I use my own credentials instead? I want to focus on hive module example

Reply:
I don't mind. Let them be as is if you like :+1: